querybook/webapp/components/AIAssistant/AICommandBar.tsx

```diff
@@ -86,8 +84,23 @@ export const AICommandBar: React.FC<IQueryCellCommandBarProps> = forwardRef(
         } = useCommand(command, commandRunner);
 
         useEffect(() => {
-            setTables((tables) => uniq([...tablesInQuery, ...tables]));
-        }, [tablesInQuery]);
+            setTables((tables) => {
+                if (mentionedTables.length > 0) {
+                    return mentionedTables;
+                } else {
+                    return uniq([...tablesInQuery, ...tables]);
+                }
+            });
+        }, [tablesInQuery, mentionedTables]);
+
+        useEffect(() => {
+            if (!query && mentionedTables.length === 1) {
+                onUpdateQuery(
+                    `SELECT\n  *\nFROM\n  ${mentionedTables[0]}\nLIMIT\n  10`,
```

Comment:
its easier to do `formatQuery("SELECT * FROM table LIMIT 10")`

Reply:
formatQuery doesn't return anything, but will change the editor content in-place

Reply:
use `import { format } from 'lib/sql-helper/sql-formatter';`

Reply:
calling onFormatQuery after updating query instead